app/utils.ts

```
@@ -3,12 +3,27 @@ import { showToast } from "./components/ui-lib";
 import Locale from "./locales";
 import { RequestMessage } from "./client/api";
 import { ServiceProvider } from "./constant";
+import { useAccessStore } from "./store";
 // import { fetch as tauriFetch, ResponseType } from "@tauri-apps/api/http";
 import { fetch as tauriStreamFetch } from "./utils/stream";
 import { VISION_MODEL_REGEXES, EXCLUDE_VISION_MODEL_REGEXES } from "./constant";
 import { useAccessStore } from "./store";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'useAccessStore'. Consider to delete it or rename it.

'useAccessStore' is defined here:


(lint/suspicious/noRedeclare)